index.html

```diff
@@ -421,7 +421,7 @@
+,{"system":"Oxla","date":"2024-04-09","machine":"c6a.4xlarge, 500gb gp2","cluster_size":1,"comment":"Ingests data only from non-compressed cvs.","tags":["C","analytical","somewhat PostgreSQL compatible"],"load_time":497.815,"data_size":17394972923,"result":[[0.046851,0.02652,0.02648],[0.129717,0.010132,0.010436],[0.024405,0.017653,0.015578],[1.038988,0.026278,0.025259],[0.920077,0.867753,0.858069],[1.193297,1.094777,1.088436],[0.024024,0.015385,0.014366],[0.021601,0.015406,0.013076],[1.523167,1.323168,1.30051],[1.906564,1.755886,1.844895],[0.190242,0.1447,0.136949],[0.221494,0.159692,0.160614],[0.836624,0.8148,0.813433],[1.664004,1.635831,1.621079],[0.887504,0.885837,0.875387],[1.070728,0.938593,1.021919],[2.685652,2.644121,2.647854],[2.667017,2.658392,2.628829],[4.65067,4.626599,4.701596],[0.105851,0.080334,0.05596],[0.722496,0.705024,0.738724],[0.765263,0.767057,0.739108],[1.877141,1.390639,1.434695],[16.211858,3.861157,3.897472],[0.112136,0.099527,0.097523],[0.121357,0.111725,0.112716],[0.157924,0.150043,0.148599],[0.698874,0.674169,0.685363],[51.293703,51.584485,51.114784],[0.114097,0.099482,0.102921],[0.437985,0.415509,0.408477],[0.720325,0.674251,0.681977],[89.73327,81.154516,76.413726],[7.202697,6.053296,6.002008],[5.97555,5.824706,6.135259],[0.68644,0.655987,0.6812],[0.064488,0.06473,0.062154],[0.023134,0.029644,0.023701],[0.039028,0.05021,0.04079],[0.152121,0.144167,0.132704],[0.01652,0.012679,0.009873],[0.018828,0.030098,0.021353],[0.017781,0.017341,0.015782]],"source":"oxla/results/c6a.4xlarge.json"}
```

Comment:
No need to update index.html manually, this is done by a GitHub bot automatically based on the files in folder `result/`.